.config/1espt/PipelineAutobaseliningConfig.yml

```
@@ -0,0 +1,21 @@
+## DO NOT MODIFY THIS FILE MANUALLY. This is part of auto-baselining from 1ES Pipeline Templates. Go to [https://aka.ms/1espt-autobaselining] for more details.
```

YAMLlint (1.35.1)

[error] wrong new line character: expected \n

(new-lines)